lib/workers/repository/updates/branch-name.ts

```diff
@@ -64,6 +64,11 @@ export function generateBranchName(update: RenovateConfig): void {
         update.groupSlug = `major-${update.groupSlug}`;
       }
     }
+    if (update.updateType === 'minor' && update.separateMultipleMinor) {
+      const newMajor = String(update.newMajor);
+      const newMinor = String(update.newMinor);
+      update.groupSlug = `minor-${newMajor}.${newMinor}-${update.groupSlug}`;
```

Comment:
```suggestion
      update.groupSlug = `minor-${update.newMajor}.${update.newMinor}-${update.groupSlug}`;
```
simplify, numbers are allowed in string template